api/app/Console/Commands/HardDeleteOldUsers.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\User;
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+
+class HardDeleteOldUsers extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:hard-delete-old-users';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Hard-delete all users soft-deleted for at least 5 years';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
```

Comment:
Could you pull in the logging facade and add some logging to this?  Or maybe [regular command output](https://laravel.com/docs/10.x/artisan#writing-output) will automatically be logged if run from the scheduler?

I suggest a regular "command ran, x users hard deleted" message on every run along with a "user xxx-yyy-zzz (firstname lastname, trashed at Jan 1, 2000) hard deleted" for every deletion.

Reply:
Added some logging with [dff6504](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7593/commits/dff6504706f18d044ba6db75af13374243c55644).

Reply:
Wonderful!